### PR TITLE
fix(cli): allow specifiers of unknown media types with test command

### PR DESCRIPTION
**Pull request by caspervonb**
This makes it so that `deno test foo.md` does not end up trying to run foo.md as a module and it is no longer an error to pass unknown media types as input to `deno test`.

They'll get considered for the inline extraction phase and dropped for the isolate execution phase.

See https://github.com/denoland/deno/issues/11645

### CODE REVIEW

File: cli/tests/testdata/test/test/markdown.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+The following block should be given a js extension on extraction:
+
+```js
+console.log("js");
+```
+
+The following block should be given a ts extension on extraction:
+
+```ts
+console.log("ts");
```
Comment (bartlomieju) [inline cli/tests/testdata/test/test/markdown.md:18]: Is this file actually used anywhere?

**Review by bartlomieju — COMMENTED** [review]
Please add a plain Markdown file that has no codeblocks inside and verify that no error is returned and no tests are executed

File: cli/tests/testdata/test/test/markdown.md (hunk 1 above)
Comment (caspervonb) [reply cli/tests/testdata/test/test/markdown.md:18]: test/test... bad merge.

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks

**Pull request merged**